packages/create-mechanic/function-examples/adaptive-grid/inputs/group/style.module.css

```diff
@@ -0,0 +1,7 @@
+.root {
+  margin: 1em 0;
+}
+
+.toggle {
+  margin: 1em 0;
+}
```

Comment:
Should we box the group's content off a bit more visually? I had a hard time understanding what the custom component does initially.

Reply:
Any suggestions? I added a bit of padding so the nested relationship is more clear.

Reply:
Thanks for this! I did a tweak to be closer to this.